src/sparseml/transformers/text_classification.py

```diff
@@ -464,13 +486,27 @@ def main():
     )
 
     # Preprocessing the datasets
+    if data_args.input_column_names is not None:
+        if "," in data_args.input_column_names:
+            # two input columns
+            columns = data_args.input_column_names
+            if len(columns) != 2:
+                raise ValueError(
+                    "input_column_names may only specify up to two columns "
+                    f"{len(columns)} provided: {columns}"
+                )
+            sentence1_key, sentence2_key = columns
+        else:
+            # one input column
+            sentence1_key = data_args.input_column_names
+            sentence2_key = None
```

Comment:
In the lines following this one, it seems that the keys get overwritten if a task name is set. Is this the expected behavior?

Reply:
switching below to elif

Reply:
input_column_names should take priority